### ISSUE TITLE
[Accessibility] A11y_.Net Editor: A user is unable to fold the code in the editor using keyboard

### ISSUE THREAD

**Issue by ChetanDumbala** [dotnet/try#121]
**Environment Details**
https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
Browser: Edge (41.16299.402.0)
Chrome (67.0.3396.87)

**Additional Details:**
MAS Violated: MAS2.1.1

**Repro steps:**
Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
Tab till “Enter focus mode” and press enter
Write a Sample program
Now press F1 and tab till Fold and press enter
Verify Fold action is getting performed or not.

**Actual Result:**
A user is unable to fold the code in the editor using keyboard

**Expected Result:**
A user should be able to fold the code using either shortcut or navigating to Fold in command Palette

**User Impact:**
A disabled user wouldn't be able to avail the functionality.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[A User is unable to fold the code in the editor.pptx](https://github.com/dotnet/try/files/2191629/A.User.is.unable.to.fold.the.code.in.the.editor.pptx)

**Comment by LadyNaggaga** [comment]
Opened an issue an issue with [monaco-editor](https://github.com/Microsoft/monaco-editor/issues/1168)

**Comment by LadyNaggaga** [comment]
**Update**
Issue has been flagged and assigned in the Monaco repo
https://github.com/Microsoft/monaco-editor/issues/1168

**Comment by LadyNaggaga** [comment]
1.  Importance of experience 
A user should be able to fold the code using either shortcut or navigating to Fold in command Palette
2. Quality of Experience (High or Low Impact). Describe user impact and workarounds.
3. [Link to AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/93711e75-9cb7-4061-b1e9-53177a087cae) entry where issue has been documented in conformance documentation

4. Any alternate solutions or technologies evaluated
5. Ensure all evidence for hand-off to external team is provided in the bug
